tests/test_readme.py

```diff
@@ -0,0 +1,59 @@
+import pathlib
+import re
+import subprocess
+
+import pytest
+
+readme_path = pathlib.Path(__file__).parent.parent / 'README.md'
+readme = readme_path.read_text()
+
+template = r'''
+<!-- test codeblock contains output of `{command}` -->
+```
+{output}```
+'''
+pattern = template.format(
+    command=r"(?P<command>.+?)",
+    output=r"(?P<output>.+?)",
+)
+pattern = re.compile(pattern, re.DOTALL)
+matches = list(pattern.finditer(readme))
+
+
+@pytest.mark.parametrize(
+    argnames=['command', 'expected'],
+    argvalues=[match.groups() for match in matches],
+    ids=[match.group('command') for match in matches],
+)
+def test_readme_codeblock_contains_output_from(command, expected):
+    """
+    If this test fails, ensure that codeblocks in README.md have the correct
+    output. To enable this check for output in a codeblock, use the following
+    construct:
+
```

Comment:
Not necessary for merging, but I'm not sure what you mean by "codeblocks" here

Reply:
These things (https://help.github.com/articles/creating-and-highlighting-code-blocks/) in the markdown README for this repo.

Reply:
I see, thanks.  I think I'm not exactly sure what this project is/what the goal of this project is so perhaps my comments don't make too much sense

Reply:
My bad! I should have given you a bit more context. This repository creates the `manubot` command line program written in Python. The README contains some codeblocks which are supposed to contain the output produced by running a specific manubot command, such as `manubot --help`. This PR implements a test to check that the output in the README codeblocks is what it should be given the current manubot program. Basically, it's helpful to remind someone they forgot to update the README. It also implements a script to update the README for you.